### PR TITLE
[v6r18] Fixes for MySQL 5.7

### PR DESCRIPTION
**Pull request by chaen**
For the FTSHistoryView.
And a small fx for dirac-install to allow local md5 files

For task #3293

### CODE REVIEW

File: Core/scripts/dirac-install.py [hunk 1]
```diff
@@ -318,7 +318,12 @@ def __loadCFGFromURL( self, urlcfg, checkHash = False ):
       self.__cfgCache[ urlcfg ] = cfg
       return S_OK( cfg )
     try:
-      md5Data = urlretrieveTimeout( urlcfg[:-4] + ".md5", timeout = 60 )
+      md5path = urlcfg[:-4] + ".md5"
+      if os.path.exists( md5path ):
+        with open( md5path, 'r' ) as md5File:
```
Comment (fstagni) [inline Core/scripts/dirac-install.py]: Actually, no, this file should be possibly read by python 2.6, so don't use `woth`

Comment (fstagni) [inline Core/scripts/dirac-install.py]: Up!

Comment (chaen) [inline Core/scripts/dirac-install.py]: oups, sure I can change it, but the rest of the code also uses this convention!

Comment (andresailer) [inline Core/scripts/dirac-install.py]: I think the problem was with python 2.4, which was on SL5

**Pull request merged**